quanta-t41/config.json

```diff
@@ -35,7 +35,6 @@
     },
     "workflowRoot": "workflows",
     "taskRoot": "tasks",
-    "httpTemplateRoot": "templates",
-    "httpStaticRoot": "static"
```

Comment:
@zyoung51 : I tested, if I keep nothing change for quanta-t41 folder, the generated skupack will fail:
```
curl -X POST --data-binary @./tarballs/quanta-t41_bb.tar.gz localhost:8080/api/1.1/skus/pack
{"error":"Failed to serve file request:invalid JSON in config.json: invalid value for httpStaticRoot in config.json"}
```
But if I remove it from config.json, insert skupack will success.

Meanwhile, I don't see clear word about the `httpStaticRoot` is required in the readme, do you mean that to use the quanta-t41 FW update, those BIOS/BMC image must be put at the `static` folder? If so, we'd better to create an empty folder `static` in the skupack.